pages/index.vue

```diff
@@ -1,22 +1,123 @@
+                  </h3>
+                  <p class="text-sm text-zinc-500 line-clamp-2 group-hover:text-zinc-400 transition-colors duration-200">
+                    {{ game.mShortDescription }}
+                  </p>
+                </div>
+                <div>
+                  <NuxtLink
+                    :to="`/store/${game.id}`"
+                    class="inline-flex items-center gap-x-1.5 rounded-md bg-zinc-800 px-2.5 py-1.5 text-sm font-semibold text-white shadow-sm hover:bg-zinc-700 hover:scale-105 transition-all duration-200"
+                  >
+                    View Details
+                    <ArrowRightIcon class="h-4 w-4 group-hover:translate-x-0.5 transition-transform duration-200" />
+                  </NuxtLink>
+                </div>
+              </div>
+            </div>
+          </div>
         </div>
+      </div>
+
+      <!-- Account Overview -->
+      <div class="space-y-4">
+        <h2 class="text-xl font-semibold font-display text-zinc-100">
+          Account Overview
+        </h2>
+        <dl class="grid grid-cols-1 gap-4">
+          <div class="bg-zinc-900/50 px-4 py-5 rounded-lg sm:p-6 hover:bg-zinc-900/70 hover:shadow-lg hover:shadow-zinc-900/20 transition-all duration-200">
+            <dt class="text-sm font-medium text-zinc-400">Games in Library</dt>
+            <dd class="mt-1 flex items-baseline justify-between md:block lg:flex">
+              <div class="flex items-baseline text-2xl font-semibold text-zinc-100">
+                {{ games.length }}
+                <span class="ml-2 text-sm font-medium text-zinc-400">games</span>
+              </div>
+            </dd>
+          </div>
+          <div class="bg-zinc-900/50 px-4 py-5 rounded-lg sm:p-6 hover:bg-zinc-900/70 hover:shadow-lg hover:shadow-zinc-900/20 transition-all duration-200">
+            <dt class="text-sm font-medium text-zinc-400">Account Type</dt>
+            <dd class="mt-1 flex items-baseline justify-between md:block lg:flex">
+              <div class="flex items-baseline text-2xl font-semibold text-zinc-100">
+                {{ user?.admin ? 'Administrator' : 'User' }}
+              </div>
+            </dd>
+          </div>
+          <NuxtLink 
+            to="/account"
+            class="block w-full text-center rounded-md bg-zinc-800 px-3 py-2 text-sm font-semibold text-white shadow-sm hover:bg-zinc-700 hover:scale-105 transition-all duration-200"
+          >
+            Manage Account Settings
+          </NuxtLink>
+        </dl>
+      </div>
     </div>
+  </div>
 </template>
 
 <script setup lang="ts">
-useHead({
-    title: "Home",
-});
+import { ArrowRightIcon, PlusIcon } from '@heroicons/vue/24/outline'
+import { type Game } from '@prisma/client'
 
+const user = useUser();
 const headers = useRequestHeaders(["cookie"]);
-const games = await $fetch("/api/v1/store/recent", { headers });
+
+// Fetch data
+const games = await $fetch<Game[]>("/api/v1/store/recent", { headers });
```

Comment:
This endpoint should be left as-is, and a separate endpoint should be used for recently updated. Essentially, there shouldn't be any client side logic to sort this stuff.